### PR TITLE
Update QuickstartDeployPortal.md

### PR DESCRIPTION
**Pull request by zxue**
Clarify a few settings with examples, etc.

## Description
Describe the changes in this PR.

## Related issues
Addresses [issue #].

## Testing
Describe how this change was tested.

### CODE REVIEW

File: docs/QuickstartDeployPortal.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Below is a table that describes all of the parameters in the custom deployment t
...
 |Fhir Version|**Yes**|Select the FHIR version (STU3, R4, or R5) that you want your FHIR Server to be on.|
-|Enable Export|No|Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. You will need owner or user-administrator permissions for this. If you set this to false, you can change it to true after deployment but you will have to create and link the storage account yourself.|
+|Enable Export|No|Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. If you set this to false, you can change it to true after deployment and add the storage account value. More info on the storage setting, check [Bulk Export](https://github.com/microsoft/fhir-server/blob/master/docs/BulkExport.md). Note that you will need owner or user-administrator permissions for this. If you encounter any storage permission or role assignment related issue, a quick workaround is that you set the option to false and ask your administrator to help update the setting later.|
```
Comment (CaitlinV39) [inline docs/QuickstartDeployPortal.md:42]: Just a slight rewording - 

If you set this to false, you can change it to true after deployment and add the storage account value. For more info on the storage setting, check [Bulk Export](https://github.com/microsoft/fhir-server/blob/master/docs/BulkExport.md). Note that you will need owner or user-administrator permissions for this. If you encounter any storage permission or role assignment related issues, a quick workaround is to set **Enable Export** to false and ask your administrator to help update the setting later.|

**Review by CaitlinV39 — APPROVED** [review]
@zxue I made a small suggestion on the wording for the last piece but the rest looks good to me.

**Comment by CaitlinV39** [comment]
@zxue if you make that small wording change I approved this one so you can merge.

File: docs/QuickstartDeployPortal.md (hunk 1 above)
Comment (zxue) [reply docs/QuickstartDeployPortal.md:42]: I just updated it.

Comment (CaitlinV39) [reply docs/QuickstartDeployPortal.md:42]: Sounds good! you should be able to squash/merge but let me know if you can't.

**Pull request merged**